### ISSUE TITLE
Doesn't find anything

### ISSUE THREAD

**Issue by maurelian** [alexstep/SublimeLinter-contrib-solium#3]
## Issue

Things that solium finds in the termainl are not reported by this package.

## To reproduce

```
pragma solidity ^0.4.0;
contract A {
  function foo() {
    assembly {
    }
   } 
}
```

Lint this using solium on the command line, you'll get:

/Users/primary/temp/insecure.sol
  5:2    error    No visibility specified explicitly for foo function.    security/enforce-explicit-visibility
  7:4    error    Avoid using Inline Assembly.                            security/no-inline-assembly
  5:2    error    No visibility specified explicitly for foo function.    security/enforce-explicit-visibility
  7:4    error    Avoid using Inline Assembly.                            security/no-inline-assembly

Lint it in sublime, you'll get:

No lint results.

I do have a .soliumrc file. 

Thanks for your help!

**Comment by maurelian** [comment]
resolved by installing from this repo.
